packages/origin-device-registry-irec-local-api/src/device/device.module.ts

```diff
@@ -0,0 +1,16 @@
+import { Module } from '@nestjs/common';
+import { CqrsModule } from '@nestjs/cqrs';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { DeviceController } from './device.controller';
+import { Device } from './device.entity';
+import { DeviceService } from './device.service';
+import { ValidateDeviceOwnershipCommandHandler } from './handlers/validate-device-ownership.handler';
+
+@Module({
+    imports: [TypeOrmModule.forFeature([Device]), CqrsModule],
+    providers: [DeviceService, ValidateDeviceOwnershipCommandHandler],
+    exports: [DeviceService],
+    controllers: [DeviceController]
+})
+export class DeviceModule {}
```

Comment:
We already have `DeviceModule` in `origin-backend`. Is it safe to create modules with the same name in Nest?

Reply:
Yes, since they are located in other packages, you can name them whatever, also the old one will be removed.